test/ranking.spec.js

```diff
@@ -0,0 +1,112 @@
+import {
+    Ranking
+} from '../src/app/Ranking';
+
+describe("Ranking's logic", () => {
```

Comment:
Ogólnie z opisu testów wiadomo o co chodzi :) 
Możemy popracować nad uspójnieniem opisów - ale to chyba przyda przegadać się na spotkaniu.
Fajnie byłoby się trzymać jakiejś konwencji, ale te mi bardzo przypadają do gustu. Może taka kosmetyczna zmiana, to, zamiast przecinka możnaby pisać then - to takie standardowe i supportowane przez większość języków, bo często nazwa testu jest metodą (w JS akurat nie) no i wtedy przecinek nie pyknie :)

Reply:
Mogę poprawić, nie przyjęliśmy jednej formy, więc nie wiedziałam, do czego się odnieść ;)

Reply:
Dobra, myślę na dzisiaj zostawmy - dogadamy coś na spotkanku. Jak dla mnie to i tak bardzo porządnie do tego podeszłaś :)